resources/js/components/dashboard/components/Sponsors.tsx

```diff
@@ -154,7 +153,7 @@ class Sponsors extends Component<IDashboardPropsWithRouter, ISponsorState> {
                     sponsors = sponsors.filter(this.onlyUnique);
                     this.setState({
                         sponsors: sponsors.sort((a, b) => (a.name > b.name) ? 1 : -1),
-                        sponsorLive: this.props.user.type == "admin",
+                        sponsorLive: this.props.user.type != "admin",
```

Comment:
Is this actually needed - by my understanding `admins` won't be seeing sponsors, right?